src/app/base/components/heuristic-statuses/heuristic-statuses.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'kern-heuristic-statuses',
+  templateUrl: './heuristic-statuses.component.html',
+  styleUrls: ['./heuristic-statuses.component.scss']
+})
+export class HeuristicStatusesComponent implements OnInit {
+
+  @Input() status: string;
+  @Input() tooltipPosition: string = 'tooltip-bottom';
+
+  dataTip: string;
+  statusName: string;
+  color: string;
+
+  constructor() { }
+
+  ngOnInit(): void {
```

Comment:
i think this is something that can change on runtime so onChanges might be a better idea